<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="composerGraphs">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_11">
      <layout class="QVBoxLayout" name="main_reconstruct"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="control_sampling">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="sliderLabel_2">
       <property name="text">
        <string>fmax[HZ]: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fmaxInput">
       <property name="maximum">
        <number>9000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hide_secondary">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Toggle Secondary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>10</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 0);</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sampling_slider">
       <property name="maximum">
        <number>12</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sliderLabel">
       <property name="text">
        <string>0 fmax</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
